src/webSrap/webScrap.py

```diff
@@ -0,0 +1,22 @@
+import argparse
```

Comment:
module pour web scrapping
usage: python webScrap.py --output data.json

Reply:
or
`python chat-bot do scrap -o|--output <path-to-file>` 👀